internal/context.go

```diff
@@ -106,6 +114,45 @@ func ClearContext() string {
 	return previous
 }
 
+// LoginContext login to a Qlik Sense Enterprise and sets the X-Qlik-Session as a cookie
+func LoginContext(tlsClientConfig *tls.Config, contextName string) {
+	userName := viper.GetString("user")
+	password := viper.GetString("password")
+
+	handler := NewContextHandler()
+	var context *Context
+
+	if contextName == "" {
+		context = handler.GetCurrent()
+	} else {
+		context = handler.Get(contextName)
+	}
+
+	if context == nil {
+		log.Fatalf("context '%s' wasn't found.\n", contextName)
```

Comment:
This error message might be confusing in the case where there is no current context. Perhaps check different errors depending on whether there is a `contextName` or not.

Reply:
Great input - thanks